R/plot.R

```diff
@@ -0,0 +1,323 @@
+#' @title Static rasterly graphics
+#' @description Render a \code{rasterly} object. 
+#' This object can involve axes, legend, etc to better convert graphics to data.
+#' @name static
+#' @param rasterlyObj A \code{rasterly} object.
+#' @param xlim The x limits (x1, x2) of the plot.
+#' @param ylim The y limits (x1, x2) of the plot.
+#' @param xlab A y label for the plot.
+#' @param ylab A x label for the plot.
+#' @param main A title for the plot.
+#' @param sub A sub title for the plot.
+#' @param interpolate A logical value indicating whether to linearly interpolate the image
+#' (the alternative is to use nearest-neighbour interpolation, which gives a more blocky result).
+#' @param axes Show axes or not.
+#' @param legend Show legend or not.
+#' @param legendlabel The label of legend.
+#' @param legendlayer The legend represents which layer in \code{rasterly} object. Default is 1.
```

Comment:
@z267xu I would call this `legend_layer` to match the syntax of other arguments. I have suggested that `1` represents the uppermost layer, but if this is wrong, please correct me! 🙂 

```suggestion
#' @param legendlayer Numeric. Specify the layer level within the \code{rasterly} object. The default layer level is `1`, which represents the uppermost layer.
```

Reply:
Yes, it is what I want!